docs/resources/netflow_exporter_policy.md

```diff
@@ -0,0 +1,153 @@
+---
+# Documentation generated by "gen/generator.go"; DO NOT EDIT.
+# In order to regenerate this file execute `go generate` from the repository root.
+# More details can be found in the [README](https://github.com/CiscoDevNet/terraform-provider-aci/blob/master/README.md).
+subcategory: "Tenant Policies"
+layout: "aci"
+page_title: "ACI: aci_netflow_exporter_policy"
+sidebar_current: "docs-aci-resource-aci_netflow_exporter_policy"
+description: |-
+  Manages ACI Netflow Exporter Policy
+---
+
+# aci_netflow_exporter_policy #
+
+Manages ACI Netflow Exporter Policy
+
+
+
+## API Information ##
+
+* Class: [netflowExporterPol](https://pubhub.devnetcloud.com/media/model-doc-latest/docs/app/index.html#/objects/netflowExporterPol/overview)
+
+* Supported in ACI versions: 2.2(1k) and later.
+
+* Distinguished Name Formats:
+  - `uni/infra/exporterpol-{name}`
+  - `uni/tn-{name}/exporterpol-{name}`
+
+## GUI Information ##
+
+* Locations:
+  - `Tenants -> Policies -> NetFlow -> NetFlow Exporters`
+  - `Fabric -> Access Policies -> Policies -> Interface -> NetFlow -> NetFlow Exporters`
+
+## Example Usage ##
+
+The configuration snippet below creates a Netflow Exporter Policy with only required attributes.
+
+```hcl
+
+resource "aci_netflow_exporter_policy" "example_tenant" {
+  parent_dn = aci_tenant.example.id
+  name      = "netfow_exporter"
+}
+
+```
+The configuration snippet below shows all possible attributes of the Netflow Exporter Policy.
+
+!> This example might not be valid configuration and is only used to show all possible attributes.
+
+```hcl
+
+resource "aci_netflow_exporter_policy" "full_example_tenant" {
+  parent_dn      = aci_tenant.example.id
+  annotation     = "annotation"
+  description    = "description"
+  dscp           = "AF11"
+  dst_addr       = "2.2.2.1"
+  dst_port       = "https"
+  name           = "netfow_exporter"
+  name_alias     = "name_alias"
+  owner_key      = "owner_key"
+  owner_tag      = "owner_tag"
+  source_ip_type = "custom-src-ip"
+  src_addr       = "1.1.1.1/10"
+  ver            = "v9"
+  annotations = [
+    {
+      key   = "key_0"
+      value = "value_1"
+    }
+  ]
+  tags = [
+    {
+      key   = "key_0"
+      value = "value_1"
+    }
+  ]
+}
+
+```
+
+All examples for the Netflow Exporter Policy resource can be found in the [examples](https://github.com/CiscoDevNet/terraform-provider-aci/tree/master/examples/resources/aci_netflow_exporter_policy) folder.
+
+## Schema ##
+
+### Required ###
```

Comment:
same comments as datasource apply here, but should be fixed at same place so not doing twice